database/migrations/2017_01_28_222114_remove_viewed_at_from_contacts.php

```diff
@@ -14,6 +14,10 @@ class RemoveViewedAtFromContacts extends Migration
     public function up()
     {
         Schema::table('contacts', function (Blueprint $table) {
+            if (!$table->hasColumn('viewed_at')) {
+                return;
+            }
+
             $table->dropColumn(
                 'viewed_at'
             );
```

Comment:
If prefer to write it:
```suggestion
            if ($table->hasColumn('viewed_at')) {
                $table->dropColumn('viewed_at');
            }
```